gems/quilt_rails/docs/api.md

```diff
@@ -241,19 +241,14 @@ The `Performance::Reportable` mixin uses [https://github.com/Shopify/statsd-inst
 
 #### `quilt_rails:install`
 
-This is equivalent to running `rails generate sewing_kit:install` and `rails generate quilt:install`
+This is equivalent to running `rails generate sewing_kit:install` (after installing sewing_kit gem) and `rails generate quilt:install`
```

Comment:
If we are expecting to invoke `sewing_kit:install` here we could just add a dependency on `sewing_kit` and constrain it to the upcoming release.